tests/statistics_test.sh

```diff
@@ -67,6 +72,22 @@ function statistics_Test
   output=$(statistics --invalid-option)
   msg="Invalid parameter: --invalid-option"
   assertEquals "($LINENO)" "$msg" "$output"
+
+  output=$(statistics --day not_a_day 2> /dev/null)
```

Comment:
Double-quotes

Reply:
Where in `output=$(statistics --day not_a_day 2> /dev/null)` exactly?

Reply:
My bad... ignore my comment here.

Reply:
Ignore my previous comment.